docs/audits/cumulative-ad-shift.md

```diff
@@ -0,0 +1,24 @@
+# Reduce ad layout shift
+
+## Overview
+
+Measures the total amount of user-visible layout shift that were caused by or
+happened near ads.
```

Comment:
Does this mean that the shift this metric measures can't always be directly attributed to ads? If so I think we should have some more details about the methodology used in the more info section.

Also suggest rewording to "...shift caused by or occurring near ads".

Reply:
Went with  "...shift caused by or occurred near ads".

We can't distinguish cases where ads caused the shift or an ad shifted by some non-ad element.

Reply:
It should be "Measures the total amount of user-visible layout shift caused by or occurring near ads."

How would a user determine whether an ad is actually at fault or not, then?

Reply:
Updated docs.

It's an opaque number. The plan is to follow up with more audits for attribution. But this metric lets publishers track the impact of changes locally and track the metric over time. Even for ad-caused layout shift, this number wouldn't show the fault (e.g. GPT, some first party script, the ad format, or some other ad script)

Reply:
That's fine, we can flesh the attribution part out later. We still need to be clear about what this metric is measuring in the meantime, though. I think a note in more info about how we're measuring this (layout instability API?) and known limitations of that approach would suffice.

Reply:
PTAL

Reply:
LGTM, thanks